Schema/V1/metadatakatalog.xsd

```diff
@@ -1539,6 +1539,13 @@
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="erSkjermet">
```

Comment:
Fjerne gamle skjermetObjekt for å hindre den i bli brukt ved et uhell?

Reply:
Tatt bort nå 👍

Reply:
Da ser denne OK ut.